run-spill-build/run_spill_build.sh

```diff
@@ -95,8 +97,16 @@ fi
 LIBTG4EVENT_DIR=${LIBTG4EVENT_DIR:-libTG4Event}
 
 run root -l -b -q \
-    -e "gSystem->Load(\"$LIBTG4EVENT_DIR/libTG4Event.so\")" \
-    "overlaySinglesIntoSpillsSorted.C(\"$nuInFile\", \"$rockInFile\", \"$spillFile\", $ND_PRODUCTION_INDEX, $ND_PRODUCTION_NU_POT, $ND_PRODUCTION_ROCK_POT, $ND_PRODUCTION_SPILL_POT, $ND_PRODUCTION_SPILL_PERIOD, $ND_PRODUCTION_REUSE_ROCK)"
+     -e  "gSystem->Load(\"libPhysics.so\"); \
+          gSystem->Load(\"libEG.so\"); \
+          gSystem->Load(\"/opt/generators/genie/lib/libGFwMsg.so\"); \
+          gSystem->Load(\"/opt/generators/genie/lib/libGFwReg.so\"); \
+          gSystem->Load(\"/opt/generators/genie/lib/libGFwParDat.so\"); \
+          gSystem->Load(\"/opt/generators/genie/lib/libGFwAlg.so\"); \
+          gSystem->Load(\"/opt/generators/genie/lib/libGFwUtl.so\"); \
```

Comment:
- Remove the hardcoded path of the GENIE libraries
- Check if GENIE library path is already defined
- if not defined, define it
- use it in the script when loading libraries (e.g. `gSystem->Load(\"$LIBGENIE_DIR/libGFwMsg.so\");` or something similar)